Linking to lvm2app needs -ldl if it's built with --as-needed

---
--- liblvm/liblvm2app.pc.in
+++ liblvm/liblvm2app.pc.in
@@ -7,5 +7,5 @@
 Description: lvm2 application library
 Version: @LVM_MAJOR@.@LVM_LIBAPI@
 Cflags: -I${includedir} 
-Libs: -L${libdir} -llvm2app
+Libs: -L${libdir} -llvm2app -ldl
 Libs.private: -ldevmapper
